app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt

```diff
@@ -90,6 +92,11 @@ class SearchFragment : Fragment() {
         val thisActivity = activity
         if (thisActivity is MainActivity) searchView = SearchView(thisActivity.supportActionBar?.themedContext)
         MenuItemCompat.setActionView(searchItem, searchView)
+        searchViewModel.queryText.observe(this, Observer {
+            searchItem.expandActionView()
+            searchView.setQuery(searchViewModel.queryText.value, false)
```

Comment:
No

Reply:
What exact changes are needed to made her sir? @iamareebjamal

Reply:
Can you tell me what is the use of Observer of LiveData. By looking at this code, it is clear that you don't know how livedata works